res/database/Default/database_programmes.xml

```diff
@@ -36930,6 +36962,7 @@ Marynarka Wojenna wspierała ten film, niestety nie odniósł on takiego sukcesu
 			<title>
 				<de>An Asian Spooky Story II</de>
 				<en>An Asian Spooky Story II</en>
+				<pl>An Asian Ghost Story II</pl>
```

Comment:
If not Polish why change the English version?

Reply:
changed.